coordinator/src/main/scala/com/socrata/datacoordinator/service/Main.scala

```diff
@@ -1,35 +1,34 @@
 package com.socrata.datacoordinator.service
 
+import java.util.concurrent.{CountDownLatch, Executors, TimeUnit}
+
 import com.rojoma.json.v3.ast.{JString, JValue}
-import com.rojoma.simplearm.SimpleArm
 import com.rojoma.simplearm.util._
-import com.socrata.datacoordinator.common.{SoQLCommon, StandardDatasetMapLimits, DataSourceFromConfig}
-import com.socrata.datacoordinator.id.{UserColumnId, ColumnId, DatasetId}
-import com.socrata.datacoordinator.secondary.{DatasetAlreadyInSecondary, SecondaryLoader}
+import com.socrata.datacoordinator.common.{DataSourceFromConfig, SoQLCommon}
+import com.socrata.datacoordinator.id.{ColumnId, DatasetId, UserColumnId}
+import com.socrata.datacoordinator.secondary.{DatasetAlreadyInSecondary, SecondaryGroupInfo}
 import com.socrata.datacoordinator.truth.CopySelector
-import com.socrata.datacoordinator.truth.metadata.{SchemaField, Schema, DatasetCopyContext}
+import com.socrata.datacoordinator.truth.metadata.{DatasetCopyContext, Schema, SchemaField}
 import com.socrata.datacoordinator.util.collection.UserColumnIdSet
-import com.socrata.datacoordinator.util.{NullCache, IndexedTempFile, StackedTimingReport, LoggedTimingReport}
+import com.socrata.datacoordinator.util.{IndexedTempFile, LoggedTimingReport, NullCache, StackedTimingReport}
 import com.socrata.http.common.AuxiliaryData
 import com.socrata.http.server.curator.CuratorBroker
 import com.socrata.http.server.livenesscheck.LivenessCheckResponder
 import com.socrata.http.server.util.{EntityTag, Precondition}
-import com.socrata.soql.environment.ColumnName
 import com.socrata.thirdparty.curator.{CuratorFromConfig, DiscoveryFromConfig}
 import com.socrata.thirdparty.typesafeconfig.Propertizer
 import com.typesafe.config.{Config, ConfigFactory}
-import java.net.{InetSocketAddress, InetAddress}
-import java.util.concurrent.{CountDownLatch, TimeUnit, Executors}
-import org.apache.curator.x.discovery.{ServiceInstanceBuilder, ServiceInstance}
+import org.apache.curator.x.discovery.ServiceInstanceBuilder
 import org.apache.log4j.PropertyConfigurator
 import org.joda.time.DateTime
+
 import scala.util.Random
 
 class Main(common: SoQLCommon, serviceConfig: ServiceConfig) {
   val log = org.slf4j.LoggerFactory.getLogger(classOf[Main])
 
   def ensureInSecondary(storeId: String, datasetId: DatasetId): Unit =
-    for(u <- common.universe) {
+    for { u <- common.universe } {
```

Comment:
I think that parens might be clearer, since it's just a loop.

Though, I really that scala's `for` is overloaded.